packages/website/pages/account/payment.js

```diff
@@ -2,68 +2,121 @@
  * @fileoverview Account Payment Settings
  */
 
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { Elements, ElementsConsumer } from '@stripe/react-stripe-js';
 import { loadStripe } from '@stripe/stripe-js';
 
+import Button from '../../components/button/button.js';
 import PaymentMethodCard from '../../components/account/paymentMethodCard/paymentMethodCard.js';
 import AccountPlansModal from '../../components/accountPlansModal/accountPlansModal.js';
+// import { plans } from '../../components/contexts/plansContext';
 import AccountPageData from '../../content/pages/app/account.json';
 // import PaymentHistoryTable from '../../components/account/paymentHistory.js/paymentHistory.js';
-// import CurrentBillingPlanCard from '../../components/account/currentBillingPlanCard/currentBillingPlanCard.js';
 import AddPaymentMethodForm from '../../components/account/addPaymentMethodForm/addPaymentMethodForm.js';
+import { getSavedPaymentMethod } from '../../lib/api';
+
+// const currentPlan = plans.find(p => p.current);
+
+// const CurrentBillingPlanCard = props => {
+//   return (
+//     <div className="billing-card card-transparent">
+//       {currentPlan !== undefined && (
+//         <div key={currentPlan.title} className="billing-plan">
+//           <h4 className="billing-plan-title">{currentPlan.title}</h4>
+//           <p className="billing-plan-desc">{currentPlan.description}</p>
+//           <p className="billing-plan-limit">
+//             <span>Limit: {currentPlan.amount}</span>
+//             <span>Overage: {currentPlan.overage}</span>
+//           </p>
+//           <div className="billing-plan-amount">{currentPlan.price}</div>
+//           <div className="billing-plan-usage-container">
+//             <p className="billing-label">Current Usage:</p>
+//             <div className="billing-plan-usage">
+//               <div className="billing-plan-meter">
+//                 <span className="billing-plan-meter-used"></span>
+//               </div>
+//               100GB
+//             </div>
+//           </div>
+//         </div>
+//       )}
+//     </div>
+//   );
+// };
```

Comment:
is this to drop?

Reply:
I'll make sure this gets removed in the fast-follow ui iteration, which we cut down a bit just to get a subset of it in here. @adamalton heads up, best to just remove code than comment it out. (you can always revert a commit that removes things).